pkg/util/throttle.go

```diff
@@ -47,7 +48,7 @@ func NewFakeRateLimiter() RateLimiter {
 	return &fakeRateLimiter{}
 }
 
-func (t *tickRateLimiter) CanAccept() bool {
+func (t *tickRateLimiter) TryAccept() bool {
```

Comment:
This is private implementation.
The docs should be covered in public interface, right?